tests/functional/processes/file/test_fulfillment_process.py

```diff
@@ -0,0 +1,111 @@
+from processes import ClusterProcess, FulfillURLManifestProcess
+from model import Record, Item, Edition, Work, Link
+import json
+import os
+from sqlalchemy.orm import joinedload
+
+
+def test_fulfillment_process(db_manager, s3_manager, seed_test_data):
+
+    manifest_record_uuid = seed_test_data['manifest_record_uuid']
```

Comment:
would recommend calling this limited_access_record_uuid since we are working with limited access records for this test